src/components/Card/_card.scss

```diff
@@ -1,6 +1,8 @@
-@import '~carbon-components/scss/globals/scss/vars';
+@import '../../globals/vars';
 @import '~carbon-components/scss/globals/scss/layout';
 
+$iot-header-padding: $spacing-05;
+
 .card {
```

Comment:
Add the prefix while you're here

Reply:
I did consider that, but this "card" string itself is a prefix for a bunch of other classes (like "card--toolbar-action") due to the Sass ampersand construct: `.card { &--toolbar-action {...} }`. These classes are in turn used used by other components and _Styled Components_ so such a refactoring would spread out and I did not want to touch them at this time. Is it okay address this in a separate issue? 

I personally don't think this ampersand naming construct is a good strategy since it makes gradual refactoring more difficult.

Reply:
We can import the prefix to the js files and prepend the prefix on the styled components definition as well as the className. Searching globally in the project this is the only `.card--` usage I could find in styled components https://github.com/IBM/carbon-addons-iot-react/blob/5a0689ae45cd47a7e5dd2a787f5881ba6ea17639/src/components/Card/CardToolbar.jsx#L44-L63

There's ~10 other spots we'd need to update the className, as well as 1 spot in a selector in Card.text.jsx

Probably beyond the scope here, but it would be nice to have the prefix so consumers can reliably override the styles if they need to. If we don't include it here, we should open a new issue to track this and address when we can.

Reply:
Also I agree with you on the ampersand construct - we should avoid it's use and nesting as well to provide an easier path for maintenance and refactoring.

Reply:
I would argue that the use of ampersand makes it easier to rename elements. Change the overarching class and the rest of them are changed. When would you add a prefix to one but not all of the classes that fall under that component?

Reply:
@tay1orjones Yes, I could add the prefix to the Styled Component as well, but even better would be to move the Styled Component over to the scss file. And then this refactor would grow even bigger.

@davidicus I agree that we would want to change everything related to the card naming to be consistent. My point was that we are making it more difficult to do it gradually step by step with this construct which might lead a developer to continue using the old naming scheme instead of renaming everything if it leads to a bigger refactor that the task itself would call for. Anyway, it is not really a big deal here. Should I just go ahead and rename it all in this PR or should we do that in a separate issue and include removing the Styled Components as well?

Reply:
A separate issue is fine from my POV